requirements/dev-requirements.txt

```diff
@@ -294,6 +292,7 @@ gnureadline==8.1.2
 google-api-core==2.11.0
     # via
     #   firebase-admin
+    #   google-api-core
```

Comment:
@jingcheng16 I'm getting a test error about inconsistent requirements. Didn't you run into this recently?

Reply:
I was behind, thank you! Updated and committed new requirements files.